astropy/samp/errors.py

```diff
@@ -32,3 +32,9 @@ class SAMPProxyError(xmlrpc.Fault):
     """
     SAMP Proxy Hub exception.
     """
+
+
+class SAMPProxyTimeout(SAMPProxyError):
```

Comment:
Keep Error in the name, so it's keeps being consistent with all the other exceptions and warnings.

```suggestion
class SAMPProxyTimeoutError(SAMPProxyError):
```